frontend/packages/console-dynamic-plugin-sdk/src/index.ts

```diff
@@ -4,3 +4,5 @@ export * from './api/common-types';
 
 // Extension types
 export * from './extensions';
+
+export * from './perspective';
```

Comment:
This should go into a module that's part of the entry point which TypeScript compiler works with when generating the `dist` directory, e.g. in `src/index-lib.ts`.